lib/iris/tests/results/netcdf/netcdf_save_gridmapmulti.cdl

```diff
@@ -15,12 +15,12 @@ variables:
 		latitude_longitude:earth_radius = 6371229. ;
 	int64 longitude(longitude) ;
 		longitude:axis = "X" ;
-		longitude:units = "degrees_east" ;
+		longitude:units = "1" ;
```

Comment:
In the test these coordinates are created from numpy arrays without specifying a unit, so it defaults to '1'.